game-service/src/main/java/org/libertybikes/game/party/PartyQueue.java

```diff
@@ -17,6 +22,18 @@
     private final ConcurrentLinkedDeque<QueuedClient> waitingPlayers = new ConcurrentLinkedDeque<>();
     private final Party party;
 
+    private Metadata currentPlayersCounter = new Metadata("current_num_of_players_in_queue", // name
+                    "Current Number of Players Waiting In A Queue", // display name
+                    "Number of players that are currently waiting in a queue", // description
+                    MetricType.COUNTER, // type
+                    MetricUnits.NONE); // units
+
+    private MetricRegistry registry;
+
+    private MetricRegistry getRegistry() {
```

Comment:
(optional comment) I wonder if it would be better to have all the metrics code separated out into its own class. That way we wouldn't need this helper method in every class and instead could init all `Metadata` once and get the `MetricsRegistry` once